src/components/TimeCardPage/TimeSheet.tsx

```diff
@@ -288,8 +291,11 @@ export default function Page() {
                     </> : <></>}
                 <DateSelectorCard onDateChange={updateDateRange} date={selectedDate} />
                 
+                
             </HStack>
             {useMemo(() => renderWarning(), [selectedDate])}
+        <div>
+          <fieldset disabled={disabled}>
```

Comment:
🔧 This also disables the tabs, which we don't want to disable to allow users to look at old timesheets.